cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1075,7 +1073,7 @@ Slice::Gen::validateMetadata(const UnitPtr& u)
     knownMetadata.emplace("cpp:type", typeInfo);
 
     // Pass this information off to the parser's metadata validation logic.
-    Slice::validateMetadata(u, "cpp", knownMetadata);
+    Slice::validateMetadata(u, "cpp", std::move(knownMetadata));
```

Comment:
The one variable we want to move in this function is the big map of metadata. It's odd this std::move was missing from all compilers.